lightly/utils/scheduler.py

```diff
@@ -167,12 +171,16 @@ def __init__(
         end_value: float = 0.001,
         period: Optional[int] = None,
         verbose: bool = False,
+        warmup_start_value: float = 0.0,
+        warmup_end_value: Optional[float] = None,
     ) -> None:
         self.warmup_epochs = warmup_epochs
         self.max_epochs = max_epochs
         self.start_value = start_value
         self.end_value = end_value
         self.period = period
+        self.warmup_start_value = warmup_start_value
```

Comment:
I'm surprised this works but ok 😄

Reply:
Does it also work if I change `warmup_start_value`?

Reply:
Yes, `self.warmup_start_value` is passed later on to `cosine_warmup_schedule(warmup_start_value=self.warmup_start_value)`